app.py

```
@@ -1651,11 +1654,45 @@ def _run_consolidation_tick() -> None:
+        tick_start = time.perf_counter()
         results = scheduler.run_scheduled_tasks(
             decay_threshold=CONSOLIDATION_DECAY_IMPORTANCE_THRESHOLD
         )
         for result in results:
             _persist_consolidation_run(graph, result)
+
+            # Emit SSE event for real-time monitoring
+            task_type = result.get("mode", "unknown")
+            steps = result.get("steps", {})
+            affected_count = 0
+            sample_ids: List[str] = []
```

Ruff (0.14.10)

Local variable `sample_ids` is assigned to but never used

Remove assignment to unused variable `sample_ids`

(F841)